client/app/cloud/components/project/billing/instance-list/billing-instance-list.component.html

```diff
@@ -1,94 +1,72 @@
-                    </span>
-                    <span data-ng-if="instanceConsumption.monthlyBilling && instanceConsumption.monthlyBilling.status === 'activationPending'">
-                        {{ :: 'cpbc_hourly_instance_monthly_billing_activationPending' | translate }}
-                        <i class="fa fa-clock-o left-space-m8"></i>
-                    </span>
-                    <span class="opacity6" data-ng-if="!instanceConsumption.monthlyBilling && instanceConsumption.isDeleted" data-translate="cpbc_hourly_instance_deleted"></span>
+<oui-datagrid data-rows="$ctrl.instanceConsumptionDetails" data-ng-if="$ctrl.instanceConsumptionDetails.length > 0">
+    <oui-column data-title=":: $ctrl.instanceColName | translate" data-sortable data-property="instanceName">
+        <span data-ng-bind=":: $row.instanceName"
+              data-responsive-popover="'app/cloud/components/project/billing/details-popover/instance-details-popover.html'"
+              data-popover-class="details-popover"
+              data-popover-is-open="$ctrl.DetailsPopoverService.isCurrentDetails($row)"
+              data-popover-placement="top"
+              data-popover-append-to-body="true"
+              data-popover-trigger="none"
+              data-ng-click="$ctrl.DetailsPopoverService.setCurrentDetails($row)"></span>
+    </oui-column>
+    <oui-column data-title=":: $ctrl.instanceColTotal | translate" data-property="total" data-sortable>
+    </oui-column>
+    <oui-column data-ng-if="$ctrl.showSwitchToMonthlyBillingOption || $ctrl.showAdditionalInstanceDetails">
+        <span data-ng-if="$row.monthlyBilling && $row.monthlyBilling.status === 'ok'"
+              data-translate="cpbc_hourly_instance_monthly_billing_since"
+              data-translate-values="{ since : ($row.monthlyBilling.since | date:'shortDate') }">
+        </span>
+        <span data-ng-if="$row.monthlyBilling && $row.monthlyBilling.status === 'activationPending'">
+            {{ :: 'cpbc_hourly_instance_monthly_billing_activationPending' | translate }}
+            <i class="fa fa-clock-o left-space-m8"></i>
+        </span>
+        <span class="opacity6"
+              data-ng-if="!$row.monthlyBilling && $row.isDeleted"
+              data-translate="cpbc_hourly_instance_deleted">
+        </span>
 
-                    <!-- CONFIRM PASSING TO MONTHLY PAYMENT -->
-                    <div data-ng-show="instanceConsumption.instanceId === $ctrl.instanceToMonthly"
-                            class="text-right animate-fadeInRight-enter animate-fadeOutLeft-leave full-width cell-confirm">
+        <!-- CONFIRM PASSING TO MONTHLY PAYMENT -->
+        <div data-ng-if="$row.instanceId === $ctrl.instanceToMonthly"
+             class="text-right animate-fadeInRight-enter animate-fadeOutLeft-leave full-width cell-confirm">
+            <button class="btn btn-default"
+                    type="reset"
```

Comment:
Shouldn't be a `type="button"`?  Because I can't find any `<form>` which include this component.